go.mod

```diff
@@ -146,10 +145,10 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.4.0
 	helm.sh/helm/v3 v3.11.1
-	k8s.io/api v0.27.9
-	k8s.io/apiextensions-apiserver v0.27.9
-	k8s.io/apimachinery v0.27.9
-	k8s.io/apiserver v0.27.9
+	k8s.io/api v0.28.3
+	k8s.io/apiextensions-apiserver v0.28.3
+	k8s.io/apimachinery v0.28.3
+	k8s.io/apiserver v0.28.3
 	k8s.io/cli-runtime v0.27.9
```

Comment:
So it looks like we're upgrade the k8s modules as well to 1.28. Does this means that we're adding (partial) support to 1.28 in this PR? (I have the same question for 2.8) Does this require regression testing in Rancher to make sure that we 1.28 doesn't break anything (at least the go modules part of it)?

Reply:
Discussed this offline with @KevinJoiner . This is pulled in because of the operator changes. We also replace these imports above with the 1.27 versions: https://github.com/rancher/rancher/blob/44e905966c58ef0fd21875331aa143bed97f604b/go.mod#L28-L31, so I think that this should be ok. Happy to discuss further if you disagree.